quickstart/src/joinroom.js

```diff
@@ -226,8 +323,25 @@ async function joinRoom(token, connectOptions) {
   const logger = Logger.getLogger('twilio-video');
   logger.setLevel('debug');
 
+  if(!window.CitrixWebRTC) {
+    throw new Error('CitrixWebRTC SDK is not available yet or is not loaded properly');
+  }
+
   // Join to the Room with the given AccessToken and ConnectOptions.
-  const room = await connect(token, connectOptions);
+  const webRTCRedirections = {
+     // Custom connect options for WebRTC redirection.
+     getUserMedia: (...args) => window.CitrixWebRTC.getUserMedia(...args),
+     enumerateDevices: window.CitrixWebRTC.enumerateDevices,
+     RTCPeerConnection: window.CitrixWebRTC.CitrixPeerConnection.bind(window.CitrixWebRTC),
+     MediaStream: CustomMediaStream,
+     // Connect options required for WebRTC Unified Plan SDP semantics.
+     sdpSemantics: 'unified-plan',
+     enableDtlsSrtp: true,
```

Comment:
Are these used somewhere in the SDK? I don't see any references to them.

Reply:
Good question! I should have clarified that these are intended to be passed as RTCConfiguration.